tests/code/test_emailHandler.py

```diff
@@ -10,7 +10,7 @@
 from app.models.eventRsvp import EventRsvp
 from app.models.user import User
 from app.models import mainDB
-from app.logic.emailHandler import EmailHandler
+from app.logic.emailHandler import *

```

Comment:
The asterisk is a wild card with possible consequences later on. Not pressing, but consider importing what you need explicitly.

Reply:
I agree. In general I'm not a fan of blanket imports